completions/bash/kpod

```diff
@@ -140,6 +140,15 @@ _kpod_save() {
      _complete_ "$options_with_args" "$boolean_options"
 }
 
+_kpod_export() {
+     local options_with_args="
+     --output -o
```

Comment:
I actually want to drop buildah export, as soon as this gets merged.  We should match what docker export does.